ckan/lib/config_tool.py

```diff
@@ -235,7 +236,7 @@ def insert_new_sections(new_sections):
                         (key, existing_option.value, section)
 
             if changes_made:
-                print(changes_made)
+                log.debug(changes_made)
```

Comment:
Other CLI commands output to stdout with `click.secho`, shouldn't we do the same here? I found useful to know what was changed in the ini file when using this command. In any case if we want to use logging instead this should be probably be a `log.info()`

Reply:
This code is outside `ckan.cli` module, so I don't want to tie it to the `click`. But using `log.info` is good alternative